debian/libnx-x11-6.symbols

```diff
@@ -547,7 +547,7 @@ libNX_X11.so.6 libnx-x11-6 #MINVER#
  XcmsTekHVCWhiteShiftColors@Base 3.5.0.29
  XcmsUNDEFINEDColorSpace@Base 3.5.0.29
  XcmsVisualOfCCC@Base 3.5.0.29
- XdmcpWrap@Base 3.5.0.29
+#MISSING: 2:3.5.99.0-0~build1# XdmcpWrap@Base 3.5.0.29
```

Comment:
The XdmcpWrap() function is only used once in nx-X11/lib/X11/ConnDis.c.

XdmcpWrap() is actually a symbol of libXdmcp that used to be source-symlinked-into libX11 in the old days. Nowadays, the symbol is available via <X11/Xdmcp.h> (which already gets included in nx-X11/lib/X11/ConnDis.c).